### PR TITLE
Develop Posts and Comments CRUD operations

### PR DESCRIPTION
**Pull request by tony-tang-aligent**
For this version, I just went on to implement all the Posts and Comments CRUD operatoins.

Already finished all the user session, Posts and Comments CRUD.

I would revise the Dependency Injection usage, namespace usage and .env in later version.

Currently been working on the admin functions

### CODE REVIEW

**Comment by hellboy1975** [comment]
Just noting that the comments I've made in the other PR https://github.com/tony-tang-aligent/backend_onboard_blog/pull/1 are also relevant in this one

File: app/config/info.php
Comment (hellboy1975) [inline app/config/info.php:1]: While a file with `phpinfo()` can be useful to testing applications, it's good practice to not include such a thing in  your codebase unless there's a good reason to do so.

Comment (tony-tang-aligent) [reply app/config/info.php:1]: Sure thing, will make sure to delete it, just use it to configure my developing env

**Comment by tony-tang-aligent** [comment]
My approach is that I just focus on the functions themselves, because I am in the middle of something, so I want to revise my codebase using namespace, .env etc after I am done with all the function requirements

**Pull request closed without merging**